### DISCUSSION TITLE
Control multiple Users with one Client ID

### DISCUSSION THREAD

**Discussion by Jilence** [spotify-web-api-java/spotify-web-api-java#265] [Q&A] [answered]
Hello!

I have following problem. So, I created a Spotify-Application, for the client_id and the client_secret. Then I created a link, that return the auth code. With this code, I create the access token and refresh token. 
Now I want to control the Spotify Account, from the User, how clicked on the linked, with my client_id.
**BUT**, when I click on the link, its working, but when my friends click on the link, it didnt work. 

So is this my fault?

**Comment by dargmuesli** [comment, marked as answer]
Hello, please respect the issue template.

**Reply by dargmuesli** [reply]
I'm unable to help you at the "it didnt work" step, because I have no way to know what you mean by that.

**Reply by Jilence** [reply]
So, the problem is that I can only control my own Spotify account after authenticating (pause, skip, ...). If another person authenticates and I try to skip songs or something, nothing happens (the song is not paused or skipped).

**Reply by dargmuesli** [reply]
That sounds like a question towards the Spotify API, not towards this repository's source code. This is the right place to ask: https://community.spotify.com/t5/Spotify-for-Developers/bd-p/Spotify_Developer

The API only allows controlling the authenticated account. You cannot control the account of someone else with your personal access token.

**Reply by Jilence** [reply]
Yeah, I know. I have the access token of the user. But it doesnt work with this access token. Only my access token is working. If I have someone else's access token and try to control spotify with it, it doesn't work.

**Reply by dargmuesli** [reply]
Please head over to the link mentioned above to find an answer to that question.

**Reply by Jilence** [reply]
Okay, 1 last question. Is it possible to control the Spotify Account of a user just with the access token?

**Reply by dargmuesli** [reply]
You should be able to find the answer in Spotify's API documentation, https://developer.spotify.com/, or in their developer community forum. This repository only offers support to questions regarding its own source code.
